index.html

```diff
@@ -233,5 +258,15 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
     <hr>
    </section>
 
+   <footer class="mobile-hide">
+    <div class="social-media">
+      <a href="#"><i class="fab fa-twitter social"></i></a>
+      <a href="#"><i class="fab fa-linkedin social"></i></a>
+      <a href="#"><i class="fab fa-medium-m social"></i></a>
+      <a href="#"><i class="fab fa-github social"></i></a>
+      <a href="#"><i class="fab fa-angellist social"></i></a>
+    </div>
```

Comment:
- [ ] [REQUIRED] It will be nice to reduce the sizes of these social media icons so that your design can look more like the figma template provided.
- [ ] [REQUIRED] Kindly apply the same recommendation to the social media icons on the introduction page. Take note of the margin-top as well